scene/2d/animated_sprite.cpp

```diff
@@ -436,11 +436,10 @@ void AnimatedSprite::_notification(int p_what) {
 
 			RID ci = get_canvas_item();
 
-			Size2i s;
-			s = texture->get_size();
+			Size2i s = texture->get_size();
 			Point2 ofs = offset;
 			if (centered) {
-				ofs -= s / 2;
+				ofs -= Size2(s) / 2;
 			}
```

Comment:
Agree with the other reviewer on the master PR, this should simply be:
`Size2 s = texture->get_size();`

I have a feeling I saw someone else with a PR to change `get_size()` to return a `Size2i` or `Vector2i` but this should still work fine.

Reply:
> Agree with the other reviewer on the master PR, this should simply be:
> `Size2 s = texture->get_size();`

Ok then, I'll change that later in both PRs.


> I have a feeling I saw someone else with a PR to change `get_size()` to return a `Size2i` or `Vector2i` but this should still work fine.

Probably #52502.